### ISSUE TITLE
Can't output dest files when using 'node-bourbon' and include mixin in scss files.

### ISSUE THREAD

**Issue by nakagaw** [pxgrid/aigis#55]
はじめまして。
いろいろスタイルガイドを試していて、こちらも触ってみたのですが、
なかなか Sass でうまくいかないので、原因箇所を探っていたら、
node-bourbon の mixin で @include している箇所があると動かないみたいです。
Hologram や kss では動いていたので、対応可能かどうかだけおしえていただけるとうれしいです。

↓ Translate to my poor English :)

Hi,
I had try aigis. It looks good to me.
But, Target file couldn't get.
I was searching for the point of problem. And I found it.
I using 'node-bourbon' and included mixin in scss files.
But, Hologram or kss, I could output.
So, If you have solution, please let me know. 
Thanks.

---
OS Mac OS 10.11
gulp v3.8.11
 gulp-sass v2.1.0
 node-bourbon v4.2.3
node v4.2.2
Sass v3.4.10

**Comment by geckotang** [comment]
@nakagaw Thank you for letting me know. Do you have error log when you used aigis with node-bourbon?

**Comment by watilde** [comment]
can i have this scss file including a mixin like you said?
> I using 'node-bourbon' and included mixin in scss files.

**Comment by nakagaw** [comment]
@geckotang 
Not display error. Only displayed at console below:
```
-------- target file --------

```

**Comment by nakagaw** [comment]
@watilde I attached files. This is minimal constitution. But, same thing will happen. Thanks.
```
src
├── generic.scss
└── _buttons.scss
```
[src.zip](https://github.com/pxgrid/aigis/files/68174/src.zip)

**Comment by geckotang** [comment]
@nakagaw Sorry for my late reply. Please try to use `node-aigis@1.1.0`.

**Comment by nakagaw** [comment]
@geckotang Thanks, I've try gulp-aigis@1.1.0.
But, Getting this  'Unhandled rejection Error'. 
Do you know something solution?

```
[Info] Config File: /Users/myApp/gulp/template/config.yml
[Info] Start: Generate Styleguide
Unhandled rejection Error
    at /Users/myApp/node_modules/gulp-aigis/node_modules/node-aigis/lib/Aigis.js:85:17
    at tryCatcher (/Users/myApp/node_modules/gulp-aigis/node_modules/node-aigis/node_modules/bluebird/js/main/util.js:26:23)
    at Promise._settlePromiseFromHandler (/Users/myApp/node_modules/gulp-aigis/node_modules/node-aigis/node_modules/bluebird/js/main/promise.js:507:31)
    at Promise._settlePromiseAt (/Users/myApp/node_modules/gulp-aigis/node_modules/node-aigis/node_modules/bluebird/js/main/promise.js:581:18)
    at Promise._settlePromises (/Users/myApp/node_modules/gulp-aigis/node_modules/node-aigis/node_modules/bluebird/js/main/promise.js:697:14)
    at Async._drainQueue (/Users/myApp/node_modules/gulp-aigis/node_modules/node-aigis/node_modules/bluebird/js/main/async.js:123:16)
    at Async._drainQueues (/Users/myApp/node_modules/gulp-aigis/node_modules/node-aigis/node_modules/bluebird/js/main/async.js:133:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/myApp/node_modules/gulp-aigis/node_modules/node-aigis/node_modules/bluebird/js/main/async.js:15:14)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

---
gulp v3.8.11
gulp-sass v2.3.1
node v5.5.0
Sass v3.4.21

**Comment by geckotang** [comment]
@nakagaw can you upload the config.yml ?

**Comment by nakagaw** [comment]
@geckotang Please checking! (Some private directory name has changed. But maybe no problem..) 👍 
[config.yml.zip](https://github.com/pxgrid/aigis/files/316266/config.yml.zip)

**Comment by geckotang** [comment]
@nakagaw node-aigis@1.1.0 has a breaking change :( Perhaps, there are two problems.  

One: You need to set string type to `dest`.
https://pxgrid.github.io/aigis/docs/jp/category/Documentation/Configs/index.html

```diff
- dest:
-   -./web/styleguide
+ dest: ./web/styleguide
```

Two: I think the problem is caused by directory for templates and template files.
https://pxgrid.github.io/aigis/docs/jp/category/Documentation/Templates/index.html

```diff
- template: ./gulp/template/template
+ template_dir: ./gulp/template/template
+ template_engine: jade #or ejs or hbs
```

**Comment by nakagaw** [comment]
@geckotang Oh, I see. I'll checking documentation. Thanks.

**Comment by nakagaw** [comment]
@geckotang I've updated config.yml and retry gulp-aigis@1.1.0.

1st time, gulp compile was OK.
But, With gulp-watch package when I update a scss file. This error happen.

```
[Info] Config File: /Users/myApp/gulp/styleguide_template/aigis_config.yml
[Info] Start: Generate Styleguide
Unhandled rejection Error: Error: Error: ENOENT: no such file or directory, open '/Users/myApp/gulp/styleguide_template/Users/myApp/gulp/styleguide_template/template/index.hbs'
  at /Users/myApp/node_modules/gulp-aigis/node_modules/node-aigis/lib/Aigis.js:85:17
  at tryCatcher (/Users/myApp/node_modules/bluebird/js/main/util.js:26:23)
  at Promise._settlePromiseFromHandler (/Users/myApp/node_modules/bluebird/js/main/promise.js:507:31)
  at Promise._settlePromiseAt (/Users/myApp/node_modules/bluebird/js/main/promise.js:581:18)
  at Promise._settlePromises (/Users/myApp/node_modules/bluebird/js/main/promise.js:697:14)
  at Async._drainQueue (/Users/myApp/node_modules/bluebird/js/main/async.js:123:16)
  at Async._drainQueues (/Users/myApp/node_modules/bluebird/js/main/async.js:133:10)
  at Immediate.Async.drainQueues [as _onImmediate] (/Users/myApp/node_modules/bluebird/js/main/async.js:15:14)
  at processImmediate [as _immediateCallback] (timers.js:383:17)
```
Template's target path has became a double path.
```
/Users/myApp/gulp/styleguide_template/Users/myApp/gulp/styleguide_template/template/index.hbs
```
Relative path problem??
Please tell me if you know something..
Thanks 🙇 

Attach config.yml:
[aigis_config.yml.zip](https://github.com/pxgrid/aigis/files/329427/aigis_config.yml.zip)

**Comment by geckotang** [comment]
@nakagaw Hi, Fixed this problem on #76 . I tested on https://github.com/geckotang/aigis-issues-55  and I made sure that building a style guide is no problem. Please update gulp-aigis to latest version.

**Comment by nakagaw** [comment]
I've checked it. It's all OK.
Thank you for your supports. 🙇
